### ISSUE TITLE
Minor modif: latest_action_name

### ISSUE THREAD

**Issue by fakesecret** [RasaHQ/rasa_nlu#1834]
**Rasa Core version** :`0.7.3`

**Issue**: In trackers.py, line 234 (in _reset(self) method), self.latest_action_name = [] should be self.latest_action_name = None, preferrly adding an exaplanation in its initiation method: "latest_action_name tracks the latest executed action name"

While I was searching to show latest executed actions, I found this variable is a string instead of a list. In the initiation method, self.latest_action_name = None and in Events.ActionExecuted.apply_to, tracker.latest_action_name = self.action_name.

In summary, it is a string but never a list.
